easybuild/easyconfigs/i/imkl/imkl-2018.1.163-iimpi-2018a.eb

```diff
@@ -36,4 +36,8 @@ modextravars = {
     'MKL_EXAMPLES': '%(installdir)s/mkl/examples/',
 }
 
+modextrapaths = {
+    'PKG_CONFIG_PATH': '%(installdir)s/compilers_and_libraries_2018.1.163/linux/mkl/bin/pkgconfig',
```

Comment:
%(installdir)s/mkl/bin/pkgconfig
is shorter and version agnostic.

Reply:
@akesandgren, I fully agree.  I've pushed an update to make this change to both `eb` files.